org.mfek.glif.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.mfek.glif
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+build-options:
+  append-path: /usr/lib/sdk/rust-nightly/bin
+  env: 
+    CARGO_HOME: /run/build/MFEKglif/cargo
+    SKIA_BINARIES_URL: "file:///run/build/MFEKglif/skia-binaries.tar.gz"
+    FORCE_SKIA_BINARIES_DOWNLOAD: "1"
+finish-args: 
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=home
+command: MFEKglif
+modules:
+  - name: MFEKglif
+    buildsystem: simple
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --features=sdl2-static
+      - install -Dm755 ./target/release/MFEKglif -t /app/bin/
+      - install -Dm644 ./${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/
+      - install -Dm644 ./${FLATPAK_ID}.desktop -t /app/share/applications/
+      - install -Dm644 ./${FLATPAK_ID}.png -t /app/share/icons/hicolor/512x512/apps/
+    sources:
+      - type: git
+        url: https://github.com/MFEK/glif.git
+        commit: 8764b3e40a6dc219f3dc7db1429c58dff9795840
+      - type: file
+        url: https://github.com/rust-skia/skia-binaries/releases/download/0.62.0/skia-binaries-8cf4841aefdb295709e9-x86_64-unknown-linux-gnu-gl.tar.gz
```

Comment:
why x86_64 only?

Reply:
Compiling for aarch64 on linux is untested. As far as I know, the only thing that would need to be resolved is skia-binaries but I'm unsure. Is there a way to condition the file used on the architecture? This is my first time working on a flatpak package so I'm very new to the process

Reply:
I distinctly remember there being an aarch64-gnu distribution. Looking again, I was wrong, so aarch64 can't be supported.
https://github.com/rust-skia/skia-binaries/releases/tag/0.62.0

Reply:
why not building from source?

Reply:
I can look into building from source again, but I lost several hours trying to get it working offline. Likely a task for tomorrow though...